src/core/u-compress.c

```diff
@@ -355,9 +406,10 @@ REBSER *Decompress(
 
     // Since the initialization succeeded, go ahead and make the output buffer
     //
-    REBSER *output = Make_Binary(buf_size);
+    REBSER *output = Make_Binary(sizeof(REBSER*) + buf_size);
+    *cast(REBSER**, BIN_HEAD(output)) = output; // self-referential prefix
     strm.avail_out = buf_size;
-    strm.next_out = BIN_HEAD(output);
+    strm.next_out = BIN_HEAD(output) + sizeof(REBSER*);
 
```

Comment:
My plan to use Rebol_Malloc() and Rebol_Realloc() directly, then have COMPRESS and DECOMPRESS be the ones who Rebserize.

But this is the reason I was bringing up the gzip default...I'm going to try to reign in these routines a bit more, in a compression-oriented commit.